setup.py

```diff
@@ -53,7 +53,7 @@
     install_requires=['matplotlib>=2.0.0', 'numpy>=1.12.0', 'scipy>=0.17.0',
                       'pint', 'xarray>=0.10.7', 'enum34;python_version<"3.4"',
                       'contextlib2;python_version<"3.6"',
-                      'pooch>=0.1', 'traitlets>=4.3.0'],
+                      'pooch>=0.1', 'traitlets>=4.3.0', 'pandas'],
```

Comment:
Probably should figure out a version to support (whatever was released ~2 years ago) and update the Travis config to include this so we make sure we actually support our "minimum pandas".

Reply:
Looks like 0.21.0 is closest to two years ago. I'll add 0.24.2 to the 2.7 config for Travis, since that's the last release that supports 2.7.

Reply:
I think we should use 0.21.0 (or even 0.20.0) as the minimum. That goes back to May 2017.